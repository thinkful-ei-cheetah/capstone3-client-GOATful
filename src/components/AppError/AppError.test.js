import React from 'react';
import ReactDOM from 'react-dom';
import AppError from './AppError';

function ProblemChild() {
  throw new Error('bleh')
}
describe('AppError Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
    <AppError>
      <ProblemChild />
    </AppError>, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})
