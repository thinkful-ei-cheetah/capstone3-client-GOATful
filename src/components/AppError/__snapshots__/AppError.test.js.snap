// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppError Component renders the UI as expected 1`] = `
<div
  className="unexpected-error"
>
  <div
    className="top"
  >
    <span
      aria-label="bad news icon"
      className="emoji"
      role="img"
    >
      ðŸ¤¯
    </span>
    <h2>
      An Error Has Occurred
    </h2>
    <h4>
      Our development team has been notified. We'll get this fixed ASAP.
    </h4>
  </div>
  <div
    className="bottom"
  >
    <h4>
      try refreshing the page
    </h4>
    <i
      className="fas fa-sync-alt"
    />
  </div>
</div>
`;
